tests/test_app.py

```diff
@@ -1,8 +1,95 @@
-from tests.utils import call_from_slack
+import random
+
+import pytest
+from tests.utils import call_from_slack, get_raw_block_text, respond_interactively
 
 
 def test_help_command(chalice_app):
-    r = call_from_slack(chalice_app, ["help"], "1234", "mattias")
+    r = call_from_slack(chalice_app, "help", f"{random.randint(0, 10000)}", "mattias")
+
+    assert r["response"]["statusCode"] == 200
+    assert "Supported actions are:" in r["slack_message"][1]["json"]["text"]
+
+
+@pytest.mark.integration
+def test_empty_list(chalice_app):
+    r = call_from_slack(
+        chalice_app, "list today", f"{random.randint(0, 10000)}", "mattias"
+    )
+
+    assert r["response"]["statusCode"] == 200
+    assert "nothing to list" in r["slack_message"][1]["json"]["text"]
+
+
+@pytest.mark.integration
+def test_add_command_accepted(chalice_app):
+    user_id = f"{random.randint(0, 10000)}"
+    r = call_from_slack(chalice_app, "add vab today 8", user_id, "mattias")
+
+    assert r["response"]["statusCode"] == 200
+    assert r["slack_message"][1]["json"]["text"] == "From timereport"
+    attachments = r["slack_message"][1]["json"]["attachments"]
+    assert attachments is not None
+    attachment = attachments[0]
+    assert len(attachment["actions"]) > 0
```

Comment:
```suggestion
    assert attachment["actions"]
```
Empty list is consider falsy in python